### ISSUE TITLE
When expanding a network, limit growing width of nodes

### ISSUE THREAD

**Issue by Seanny123** [nengo/nengo_gui#555]
When I expand a network and stretch it, the nodes inside grow too. This is useful, but the width doesn't do what I would like. Instead of making a larger workspace inside the network for me to work within, I just get these silly looking really wide nodes. I think it would be useful if the nodes didn't expand width_wise past their labels.

**Comment by tcstewar** [comment]
Most of the time I want it the way it is, but I agree that sometimes it'd be nice to do just have it increase the area and not adjust anything inside it.  This is another version of the same debate we had when resizing the full window.

The only thing I can think of right now is to use SHIFT or CTRL or ALT to indicate which resizing mode you want.  So if you hold down SHIFT (or CTRL or ALT or whatever we decide) resizing doesn't affect things inside.  Unfortunately, this is a) not particularly discoverable, and b) isn't going to work when get get around to supporting touch-screen users. But it might be a nice feature to have anyway, just to try it out and see how we like it.

**Comment by celiasmith** [comment]
Yup, same debate.  It should be easy to implement, as the code for making this happen is in the code... in fact if you go to the nengo_gui preferences you can click 'Fix aspect ratio of elements on canvas resize'  and it will do this for the canvas (though not for the network).  I must admit I still like the non-resizing version, and having a hot key to switch between the two might let us figure out which is more generally preferred.

**Comment by tcstewar** [comment]
In the dev meeting, we reiterated this would be a good idea.  The first implementation to try would be that if you resize while holding down SHIFT, we don't move the internals.  This would apply to both Networks and to the netgraph as a whole (i.e. if you do something like resize the code panel).  

Note that this might interact with #705 

Also, @celiasmith would like there to be a config option that lets you switch the default.  This could be a toggle, or it could switch the default but if you do SHIFT you get one behaviour and CTRL gives the other behaviour, regardless of the default.  It's not clear what the best option is here.  But we should implement and see how it feels.
